tutorial/markdown/java/spring-boot.md

```diff
@@ -53,172 +95,364 @@ mvn package
 
 ### Database Server Configuration
 
-All configuration for communication with the database is stored in the `/src/main/resources/application.properties` file.  This includes the connection string, username, and password.  The default username is assumed to be `Administrator` and the default password is assumed to be `password`.  If these are different in your environment you will need to change them before running the application.
+the `CouchbaseConfig` class is a Spring configuration class responsible for setting up the connection to a Couchbase database in a Spring Boot application. It defines two beans:
+
+`getCouchbaseCluster()`: This bean creates and configures a connection to the Couchbase cluster using the provided hostname, username, and password.
+
+`getCouchbaseBucket(Cluster cluster)`: This bean creates a Couchbase bucket within the cluster if it doesn't already exist and returns the Bucket object associated with the specified bucket name.
 
-### Dependency Injection via DBSetupRunner Class
+### Environment Variables
 
-The quickstart code provides a CommandLineRunner called DBSetupRunner in the runners folder that wires up the Bucket and Cluster objects for dependency injection. This runner also creates the bucket, collection, scope, and indexes for the tutorial to run properly automatically when the application starts.
+You need to configure the connection details to your Couchbase Server in the application.properties file located in the src/main/resources directory.
+
+In the connection string, replace `DB_CONN_STR` with the connection string of your Couchbase cluster. Replace `DB_USERNAME` and `DB_PASSWORD` with the username and password of a Couchbase user with access to the bucket.
+
+The connection string should be in the following format:
+
+```properties
+spring.couchbase.bootstrap-hosts=couchbases://cb.fwu-oviwcpjq4v.cloud.couchbase.com
+OR
+spring.couchbase.bootstrap-hosts=localhost
+```
+
+The couchbases protocol is used for secure connections. If you are using Couchbase Server 6.5 or earlier, you should use the couchbase protocol instead.
+
+```properties
+spring.couchbase.bootstrap-hosts=DB_CONN_STR
+spring.couchbase.bucket.user=DB_USERNAME
+spring.couchbase.bucket.password=DB_PASSWORD
+```
 
 ## Running The Application
 
+### Directly on the machine
+
 At this point the application is ready, and you can run it via your IDE or from the terminal:
 
 ```shell
 mvn spring-boot:run -e -X
 ```
 
+<!-- Screeenshot -->
+
+![Spring Boot Application](./app-startup-spring-boot.png)
+
 > Note: Couchbase Server 7 must be installed and running on localhost (http://127.0.0.1:8091) prior to running the Spring Boot app.
```

Comment:
What happens if you are running Couchbase Capella?

Reply:
updated to capella